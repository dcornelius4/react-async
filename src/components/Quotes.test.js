import React from 'react';
import { shallow } from 'enzyme';
import Quotes from './Quotes';

describe('Quotes Component', () => {
  it('renders the quotes', () => {
    const quotes = [
      {
        character: 'Blender',
        quote: 'Oh crap!',
        image: '/image'
      },
      {
        character: 'Blender',
        quote: 'Where is the beer?',
        image: '/image'
      }
    ];
    const wrapper = shallow(<Quotes quotes={quotes} />);
    expect(wrapper).toMatchSnapshot();
  });
});
