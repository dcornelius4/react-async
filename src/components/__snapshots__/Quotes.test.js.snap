// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotes Component renders the quotes 1`] = `
<ul>
  <li
    key="0"
  >
    <Quote
      quote={
        Object {
          "character": "Blender",
          "image": "/image",
          "quote": "Oh crap!",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Quote
      quote={
        Object {
          "character": "Blender",
          "image": "/image",
          "quote": "Where is the beer?",
        }
      }
    />
  </li>
</ul>
`;
